frontend/src/pages/learn/when-to-take-your-pensions.tsx

```diff
@@ -0,0 +1,297 @@
+const ImportantCard: FC<ImportantCardProps> = ({ children }) => (
+  <p className="mt-4 rounded-lg bg-[#ffe8a3]/[.3] p-5">{children}</p>
+)
+
+export interface DisclaimerCardProps extends PropsWithChildren { }
+const DisclaimerCard: FC<ImportantCardProps> = ({ children }) => (
+  <p className="rounded-lg bg-[#e9f1ff]/[.7] p-5">{children}</p>
+)
+
+const WhenToTakeYourPensions: FC = () => {
+  const { t } = useTranslation('learn/when-to-take-your-pensions')
+
+  return (
+    <LearnPageLayout
+      header={t('header')}
+      learnMoreHeader={t('learn-more-heading')}
+      learnMoreLinks={[]}
+    >
+      <NextSeo title={t('header')} />
+
+      <h2 id="key-takeaways" className="h2 !mt-0">
+        {t('key-takeaways.heading')}
+      </h2>
+      <ul className="mb-5 list-disc space-y-2 pl-7">
+        <li>{t('key-takeaways.list.public-pensions')}</li>
+        <li>{t('key-takeaways.list.you-can-claim')}</li>
+        <li>{t('key-takeaways.list.to-make-a-wise')}</li>
+        <li>{t('key-takeaways.list.consider-using')}</li>
+      </ul>
+      <p>{t('key-takeaways.content-one')}</p>
+
+      <h2 id="old-age-security" className="h2">
+        {t('old-age-security.heading')}
+      </h2>
+      <p>{t('old-age-security.content-one')}</p>
+
+      <div className="flex justify-center">
+        <p>
+          {t('old-age-security.table-one.title')}
+        </p>
+      </div>
+
+      <div className="overflow-x-auto">
+        <table className="min-w-full divide-y divide-[#B9B9B9] border border-[#B9B9B9] border-collapse">
+          <thead className="bg-gray-50">
+            <tr>
+              <th className="w-1/3 px-6 py-3 text-left text-xs font-strong text-black tracking-wider border-r border-[#B9B9B9]">
+                {t('old-age-security.table-one.header-one')}
+              </th>
+              <th className="w-1/3 px-6 py-3 text-left text-xs font-strong text-black tracking-wider border-r border-[#B9B9B9]">
+                {t('old-age-security.table-one.header-two')}
+              </th>
+              <th className="w-1/3 px-6 py-3 text-left text-xs font-strong text-black tracking-wider">
+                {t('old-age-security.table-one.header-two')}
+              </th>
+            </tr>
+          </thead>
+          <tbody className="bg-white divide-y divide-[#B9B9B9]">
+            {[{
+              col1: t('old-age-security.table-one.row-one.column-one-value'),
+              col2: t('old-age-security.table-one.row-one.column-two-value'),
+              col3: t('old-age-security.table-one.row-one.column-three-value')
+            },
+            {
+              col1: t('old-age-security.table-one.row-two.column-one-value'),
+              col2: t('old-age-security.table-one.row-two.column-two-value'),
+              col3: t('old-age-security.table-one.row-two.column-three-value')
+            }].map((row, index) => (
+              <tr key={index} className={`${index === 0 ? 'bg-[#B9B9B9]' : ''}`}>
```

Comment:
Indexes should not be used as keys.

**References:**

- https://react.dev/learn/rendering-lists

Reply:
add id to each table row and not relying on array index